lib/generators/boring/vcr/install/install_generator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Boring
+  module Vcr
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds VCR to the application"
+      source_root File.expand_path('templates', __dir__)
+
+      class_option :test_framework,
+                   type: :string,
+                   alias: "-f",
+                   default: "rspec",
+                   enum: %w[rspec minitest],
+                   desc: "Tell us which test framework you are using. Default to rspec"
+
+      class_option :stubbing_libraries,
+                   type: :array,
+                   alias: "-sl",
+                   default: ['webmock'],
+                   enum: %w[webmock typhoeus faraday excon],
+                   desc: "Tell us stubbing library you want to use seprated by space. Default to webmock"
+
+      def add_vcr_gem
+        say "Adding VCR gems to Gemfile", :green
+        # TODO: Use check_and_install_gem method when it is available
+        gem 'vcr', group: :test
+      end
+
+      def add_stubbing_library_gems
+        say "Adding stubbing library gems to Gemfile", :green
+
+        options[:stubbing_libraries].uniq.each do |stubbing_library|
+          # TODO: Use check_and_install_gem method when it is available
+          gem stubbing_library, group: :test
```

Comment:
Just gem install might not be enough to properly configure gems, let's invoke generator instead.

Reply:
Documentation stated that it will work without any configuration and I have also done manual testing. so I think we don’t need to invoke generator